kernel/Kconfig

```diff
@@ -327,6 +327,14 @@ config BOOT_DELAY
 	  achieved by waiting for DCD on the serial port--however, not
 	  all serial ports have DCD.
 
+config BOOT_DELAY_RANDOM
+	bool "Randomize boot delay"
+	default n
+	help
+	  This option randomizes the bootup delay between 0 and the
+	  value of BOOT_DELAY. This is used to jitter boot-up of
```

Comment:
Instead of
`This is used to jitter boot-up of wirelessly networked devices.`
I would say
`This can be used e.g., to jitter boot-up of wireless network devices.`
to emphasize that this is just one use case.

Reply:
done